homeassistant/components/zha/core/channels/measurement.py

```diff
@@ -50,6 +58,23 @@ class OccupancySensing(ZigbeeChannel):
         AttrReportConfig(attr="occupancy", config=REPORT_CONFIG_IMMEDIATE),
     )
 
+    def __init__(self, cluster: zigpy.zcl.Cluster, ch_pool: ChannelPool) -> None:
+        """Initialize Occupancy channel."""
+        super().__init__(cluster, ch_pool)
+        if self.cluster.endpoint.manufacturer in (
+            "Philips",
+            "Signify Netherlands B.V.",
+        ) and self.cluster.endpoint.model in (
+            "SML001",
+            "SML002",
+            "SML003",
+            "SML004",
+        ):
```

Comment:
This logic is duplicated and would be really nice to have as a constant somewhere because otherwise it seems pretty likely to get out of sync.

Reply:
I've created a new ``helpers`` file in the ``channels`` folder and moved the model check in there now.